catkit/hub/cathubsqlite.py

```diff
@@ -0,0 +1,404 @@
+import sqlite3
+import json
+
+import numpy as np
+
+init_commands = [
+    """ CREATE TABLE publication (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    pub_id text UNIQUE,
+    title text,
+    authors text,
+    journal text,
+    volume text,
+    number text,
+    pages text,
+    year integer,
+    publisher text,
+    doi text,
+    tags text
+    );""",
+
+    """CREATE TABLE publication_system (
+    ase_id text REFERENCES systems(unique_id),
+    pub_id text REFERENCES publication(pub_id),
+    PRIMARY KEY (pub_id, ase_id)
+    );""",
+
+    """CREATE TABLE reaction (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    chemical_composition text,
+    surface_composition text,
+    facet text,
+    sites text,
+    coverages text,
+    reactants text,
+    products text,
+    reaction_energy real,
+    activation_energy real,
+    dft_code text,
+    dft_functional text,
+    username text,
+    pub_id text,
+    FOREIGN KEY (pub_id) REFERENCES publication(pub_id)
+    );""",
+
+    """ CREATE TABLE reaction_system (
+    name text,
+    energy_correction real,
+    ase_id text,
+    id integer,
+    FOREIGN KEY (ase_id) REFERENCES systems(unique_id),
+    FOREIGN KEY (id) REFERENCES reaction(id)
+    );"""]
+
+#index_statements = [
```

Comment:
Try to avoid commented out code in the master branch. It can make reading difficult for others and can easily be stored in a personal dev branch.